kvmagent/kvmagent/plugins/volume_cache_plugin.py

```
@@ -1043,16 +1043,29 @@ def stop(self):
-        rsp.mountPoint = pool.mount_point[MountPointInfoFields.TARGET]
+        self._fill_pool_rsp(rsp, pool)
+        return rsp
+
+    def _fill_pool_rsp(self, rsp, pool):
+        # type: (PoolRsp, PoolProcessor) -> None
+        self._fill_pool_identity(rsp, pool)
         try:
-            pool_capacity = pool.get_capacity()
-            rsp.capacity = pool_capacity.total
+            self._fill_pool_capacity(rsp, pool.get_capacity())
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)